src/tfvc/tfvc-extension.ts

```diff
@@ -110,14 +110,16 @@ export class TfvcExtension  {
                     const right: Uri = TfvcSCMProvider.GetRightResource(resource);
                     const title: string = resource.GetTitle();
 
+                    if (!right) {
+                        // TODO
+                        console.error("oh no");
+                        return;
+                    }
+
                     if (!left) {
-                        if (!right) {
-                            // TODO
-                            console.error("oh no");
-                            return;
-                        }
-                        return commands.executeCommand<void>("vscode.open", right);
+                        return await commands.executeCommand<void>("vscode.open", right);
                     }
+
                     return commands.executeCommand<void>("vscode.diff", left, right, title);
```

Comment:
Yeah, certainly to be consistent.  I'm a little under-informed about when to 'return await' on a function that returns a Promise while you're inside of a function that's async.  I've read in several places that you shouldn't actually need it as it can/does create an additional promise (the await I'll be adding here).  The act of returning should also create a promise.  But, again, I'll need to read up more to get clarity.  And, of course, there's a linting rule for this.  :-)